### ISSUE TITLE
af2guess question

### ISSUE THREAD

**Issue by kenndshi** [nrbennet/dl_binder_design#48]
Greetings. Thank you for the amazing job that You have done!
Using RFdiffusion I have generated backbones (from 10-23 amino acids long, 1000 designs) for my protein-protein complex ( orthosteric modulation, meaning that the generated backbones are in between the 2 proteins). After that, I have used fastrelax of ProteinMPNN for sequence generation. In 2 cases the complex topology didn't change much, but after running af2initial guess (as suggested in papers) the complex changed a lot.  I have a lot of models with pae<10 but the binding pocket has changed and now the models look completely different. 
Now can you please suggest a solution to this, meaning is it worth running af2guess at all, and if yes can you please tell how can I fix that problem?
Thanks in advance.

**Comment by nrbennet** [comment]
Hi, this pipeline is intended to be used for binder design where one side of the interface is rigid and fixed and the other side of the interface is designed and then predicted by AF2 in the presence of the other side. This does not sounds like the problem you are trying to solve. The scripts here should be easily modifiable to extend to your project if you wish to perform that modification.

**Comment by kenndshi** [comment]
Thanks for quick response.
I am new to protein modeling and maybe I didn't understand you so let me ask a few questions just to be clear.
1. While generating my protein-peptide-protein complex in the RFdiffusion binder design I got the pae_interaction outputs in my terminal (which seems to have not been saved). Weren't they enough to consider that my generated complex is legit?
2. Also, in the RFdiff paper they talked about PPI modulations in their experiments, but they all were allosteric inhibitions with larger binders (~100 amino acids long).  I wonder if, in my case of orthosteric modulation of 2 proteins, the RFdiffusion did the right job. ( I am pretty sure that the command that I executed for design was accurate).
3. Does proteinmpnn take into account the presence of 2 proteins in the binding pocket while designing the sequence from a helix or not?
4. If the RFdiff gave me pae scores in the initial design which seems pretty good (but unfortunately not saved, I noticed them in the terminal outputs while executing the command) why shall I run af2_initial guess after all, cuz the sequence design shall not affect the binding affinity (or shall it?)

Thanks in advance!
